### ISSUE TITLE
Gradient fill doesn't work in iOS but it works correctly in Web & Android

### ISSUE THREAD

**Issue by markgravity** [airbnb/lottie-ios#1541]
## Check these before submitting:
- [x] The issue doesn't involve an [Unsupported Feature](https://github.com/airbnb/lottie-ios/blob/master/README.md#supported-after-effects-features)
- [x] This issue isn't related to another open issue

## This issue is a:
- [x] Non-Crashing Bug (Visual or otherwise)
- [ ] Crashing Bug
- [ ] Feature Request
- [ ] Regression (Something that once worked, but doesn't work anymore)

## Which Version of Lottie are you using?

Lottie 3.3.0

## What Platform are you on? 

- [ ] MacOS
- [x] iOS

## What Language are you in?
- [x] Swift
- [ ] Objective-C

## Expected Behavior 

![Screen Shot 2022-04-05 at 15 39 19](https://user-images.githubusercontent.com/17875522/161713957-85965c66-9317-4643-aa32-224ab892bd77.png)

## Actual Behavior

![Screen Shot 2022-04-05 at 10 10 07](https://user-images.githubusercontent.com/17875522/161713993-6a8a6281-0098-4c23-ba38-9931f34908fb.png)

## Code Example
n/a

## Animation JSON

I sent the email with title
```
#1541 - Gradient fill doesn't work in iOS but it works correctly in Web & Android
```

**Comment by calda** [comment]
Could you share a copy of the animation json?

**Comment by markgravity** [comment]
@calda I sent it to brandon@withrow.io 🙏

**Comment by calda** [comment]
Can you send it to me as well please: cal.stephens@airbnb.com

**Comment by markgravity** [comment]
> cal.stephens@airbnb.com

Done it 👍

**Comment by calda** [comment]
Thanks for the report and sample json! Fixed in #1702
